form_backend/src/template_utils/validate_completed_template.js

```diff
@@ -46,6 +46,8 @@ const verify_values_are_expected_and_match_value_types = (
                 );
               })
               .value();
+          case "boolean":
+            return submitted_value;
```

Comment:
Should check if the value's a boolean, don't want to accept other values based on truthy/falsey-ness.